From 6ddf99f3275da2244370038ad87b461690f16daa Mon Sep 17 00:00:00 2001
From: Anish Trivedi <anish@freescale.com>
Date: Tue, 6 Jul 2010 15:49:16 -0500
Subject: [PATCH 166/435] ENGR00124907 MX50 linux-test: Updated test code for
 RTC

Update test code for MX50 platform.
Remove tests for SCC and SAHARA.

Signed-off-by: Anish Trivedi <anish@freescale.com>
---
 test-utils.sh                 | 1 +
 test/mxc_rtc/autorun-rtc.sh   | 2 +-
 test/mxc_sahara_test/Makefile | 2 +-
 test/mxc_scc/Makefile         | 2 +-
 test/mxc_scc2/Makefile        | 2 +-
 5 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/test-utils.sh b/test-utils.sh
index eb6b4b3..328e46f 100755
--- a/test-utils.sh
+++ b/test-utils.sh
@@ -173,6 +173,7 @@ platform()
 		MX373StackBoard )	plat=IMX37_3STACK ;;
 		MX51* )       		plat=IMX51 ;;
 		MX53* )       		plat=IMX53 ;;
+		MX50* )       		plat=IMX50 ;;
 
 		* )     ;;
 	esac
diff --git a/test/mxc_rtc/autorun-rtc.sh b/test/mxc_rtc/autorun-rtc.sh
index d15b945..ce327fe 100755
--- a/test/mxc_rtc/autorun-rtc.sh
+++ b/test/mxc_rtc/autorun-rtc.sh
@@ -9,7 +9,7 @@ STATUS=0
 
 echo "$(platform)"
 if [ "$(platform)" == IMX37_3STACK ] || [ "$(platform)" == IMX51 ] \
-    || [ "$(platform)" == IMX53 ]; then
+    || [ "$(platform)" == IMX53 ] || [ "$(platform)" == IMX50 ]; then
 	rtc_test_param=--no-periodic
 	RTC_IRQS_EXPECTED=1
 else
diff --git a/test/mxc_sahara_test/Makefile b/test/mxc_sahara_test/Makefile
index d666181..d8bbb2c 100644
--- a/test/mxc_sahara_test/Makefile
+++ b/test/mxc_sahara_test/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:= IMX21ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX37_3STACK IMX25_3STACK
+EXCLUDE_LIST:= IMX21ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX37_3STACK IMX25_3STACK IMX50
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
  OBJS= apitest
diff --git a/test/mxc_scc/Makefile b/test/mxc_scc/Makefile
index 92c0ff5..b2e6cb4 100644
--- a/test/mxc_scc/Makefile
+++ b/test/mxc_scc/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS MXC30031ADS IMX233 IMX37_3STACK IMX51"
+EXCLUDE_LIST:="IMX21ADS MXC30031ADS IMX233 IMX37_3STACK IMX51 IMX50"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/scc_test
diff --git a/test/mxc_scc2/Makefile b/test/mxc_scc2/Makefile
index cb418ec..f3b74e6 100644
--- a/test/mxc_scc2/Makefile
+++ b/test/mxc_scc2/Makefile
@@ -1,5 +1,5 @@
 # list of platforms which did not want this test case
-EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX25_3STACK IMX233"
+EXCLUDE_LIST:="IMX21ADS IMX27ADS IMX31ADS IMX31_3STACK IMX35_3STACK IMX25_3STACK IMX233 IMX50"
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
 OBJS = $(OBJDIR)/scc2_test.out
-- 
1.8.0

